autopilot/contractor.go

```diff
@@ -412,6 +436,7 @@ func (c *contractor) runContractChecks(ctx context.Context, w Worker, contracts
 	defer func() {
 		c.logger.Debugw(
 			"contracts checks completed",
+			"contractset", contractset,
```

Comment:
Is this only passed in for logging purposes? Why not log it outside the method?

Reply:
wanted it all in one line, removed it though